### PR TITLE
Update ADR index page

### PR DESCRIPTION
**Pull request by vicmgs**

### CODE REVIEW

File: assets/js/components/adr/AdrIndex.jsx [hunk 1]
```diff
@@ -1,14 +1,64 @@
...
+        </div>
+
+        <Row gutter={30} style={{ padding: 30}}>
+          <Col span={12}>
+            <div style={{ padding: 20 }}>
+              <img src={AdrIcon} style={{ height: 50, display: 'block', marginBottom: 10 }} />
+              <Text style={{ fontSize: 22, fontWeight: 600, display: 'block', marginBottom: 10 }}>Adaptive Data Rate</Text>
+              <Text style={{ fontSize: 16, fontWeight: 400, display: 'block', marginBottom: 15 }}>Adaptive Data Rate (ADR) needs to be requested by a device for this setting to have an effect.</Text>
+              <Text style={{ fontSize: 16, fontWeight: 300, display: 'block', marginBottom: 15 }}>ADR allows devices to use an optimal data rate which reduces power consumption and airtime on the network based on RF conditions. However, it is recommended to only use this setting for fixed or non-mobile devices to ensure reliable connectivity.</Text>
+              <NavLink style={{ fontSize: 16, fontWeight: 400, display: 'block', textDecoration: 'underline' }} to={"#"}>Learn more about ADR</NavLink>
```
Comment (ambuznego) [inline assets/js/components/adr/AdrIndex.jsx:53]: shouldn't this be a regular `<a>` tag since this will eventually navigate to an outside doc page?

Comment (ambuznego) [reply assets/js/components/adr/AdrIndex.jsx:53]: actually looks like we already have the doc page: https://docs.helium.com/use-the-network/console/adr/

Comment (vicmgs) [reply assets/js/components/adr/AdrIndex.jsx:53]: true that

File: assets/js/components/adr/AdrIndex.jsx [hunk 2]
```diff
@@ -1,14 +1,63 @@
-import React from 'react'
+import React, { useState } from 'react'
```
Comment (ambuznego) [inline assets/js/components/adr/AdrIndex.jsx:1]: won't need `useState` in this page I think

**Review by ambuznego — APPROVED** [review]

**Pull request merged**